services/director-v2/src/simcore_service_director_v2/modules/db/repositories/comp_runs.py

```diff
@@ -102,13 +102,13 @@ async def create(
                     .values(
                         user_id=user_id,
                         project_uuid=f"{project_id}",
-                        cluster_id=cluster_id
-                        if cluster_id != DEFAULT_CLUSTER_ID
-                        else None,
+                        cluster_id=(
+                            cluster_id if cluster_id != DEFAULT_CLUSTER_ID else None
+                        ),
                         iteration=iteration,
                         result=RUNNING_STATE_TO_DB[RunningState.PUBLISHED],
-                        started=datetime.datetime.now(tz=datetime.timezone.utc),
-                        metadata=jsonable_encoder(metadata) if metadata else None,
+                        started=datetime.datetime.now(tz=datetime.UTC),
```

Comment:
why not use arrow instead like we do in a lot of other palces?

Reply:
well that is not the purpose of this PR. I am not refactoring this at the moment.